src/pages/PlaylistPage.js

```diff
@@ -73,20 +77,49 @@ export const PlaylistPage = React.createClass({
     return (
       <Grid fluid={true}>
 
-        {/*Title with course name*/}
-        <HeadRow courseName={this.props.params.course}/>
+        {/*Title with course name and get started button*/}
+        <Row>
+          <Col xs={12} sm={6} smOffset={3}>
+            <div className={styles.headerRow}>
+              <HeadRow courseName={this.props.params.course}/>
+              <Button bsStyle="guide" className={styles.courseInfoBtn} onClick={() => this.changeState()}>
+                <Glyphicon className={styles.glyph} glyph={!this.state.showCourseInfo ? 'plus-sign' : 'minus-sign'}/>
+                Informasjon om kurset
+              </Button>
+            </div>
+          </Col>
+        </Row>
 
         <Row>
           {/*Filter desktop*/}
           <Col xsHidden>
             <Col sm={3}>{filter}</Col>
-            <Col sm={6}>{playlistsAndLessons}</Col>
+
+            {this.state.showCourseInfo ?
+              <Col sm={6}>
+                <Collapse in={this.state.showCourseInfo}>
```

Comment:
Foreslår at det lages en `courseInfo` av `<Collapse ...><CourseInfo...></Collapse>`, og at this.state.showCourseInfo bare brukes på denne.